### PR TITLE
Chinese Translation of current latest README.md

### PR DESCRIPTION
**Pull request by timothy22000**
### Summary

Chinese translation of README.md so that Chinese audiences will be able to understand and spread word about the Turing Way.

### List of changes proposed in this PR (pull-request)

* Chinese translation of README.md stored in the folder called README-translated.
* Probably addresses the CN translation part of these issues: https://github.com/alan-turing-institute/the-turing-way/issues/1335, https://github.com/alan-turing-institute/the-turing-way/issues/2051, https://github.com/alan-turing-institute/the-turing-way/issues/2107

### What should a reviewer concentrate their feedback on?

- [ ] Proofread translation
- [ ] Any broken links?
- [ ] Any english text left untranslated?
- [ ] Check formatting

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: README-translated/README-Chinese.md [hunk 1]
```diff
@@ -0,0 +1,580 @@
...
+    <td align="center"><a href="https://github.com/yaseminturkyilmaz"><img src="https://avatars3.githubusercontent.com/u/45484225?v=4?s=100" width="100px;" alt=""/><br /><sub><b>yaseminturkyilmaz</b></sub></a><br /><a href="#blog-yaseminturkyilmaz" title="Blogposts">📝</a> <a href="#ideas-yaseminturkyilmaz" title="Ideas, Planning, & Feedback">🤔</a></td>
+  </tr>
+</table>
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+
+<!-- ALL-CONTRIBUTORS-LIST:END -->
+
+This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification.
```
Comment (dingaaling) [inline README-translated/README-Chinese.md:579]: 这个项目使用 [all-contributors](https://github.com/all-contributors/all-contributors) 的规格。

File: README-translated/README-Chinese.md [hunk 2]
```diff
@@ -0,0 +1,580 @@
...
+Contributions of any kind welcome!
```
Comment (dingaaling) [inline README-translated/README-Chinese.md:580]: 欢迎所有的贡献!

**Review by dingaaling — CHANGES_REQUESTED** [review]
Looks great! Two small comments
- Punctuation edits
- Translating final English sentence at the bottom of the contributions component to Chinese

**Comment by timothy22000** [comment]
@dingaaling I really liked your suggestion so I went with that!! I'll also update and use that same translation for the CN translation of the TW book on Crowdin.

Also made the other changes now.

**Review by malvikasharan — APPROVED** [review]
Approving for merge. Thank you @timothy22000 and @dingaaling

**Comment by malvikasharan** [comment]
@all-contributors please add @timothy22000 for translation and content

**Comment by malvikasharan** [comment]
@all-contributors please add @dingaaling  for translation and review

**Pull request merged**